sqlmesh/core/macros.py

```diff
@@ -375,6 +377,15 @@ def engine_adapter(self) -> EngineAdapter:
             )
         return self.locals["engine_adapter"]
 
+    @property
+    def gateway(self) -> t.Optional[str]:
+        """Returns the gateway name."""
+        return self.var(c.GATEWAY)
+
+    def var(self, var_name: str, default: t.Optional[t.Any] = None) -> t.Optional[t.Any]:
+        """Returns the value of the specified variable, or the default value if it doesn't exist."""
+        return (self.locals.get(c.SQLMESH_VARS) or {}).get(var_name, default)
```

Comment:
maybe this method should always normalize input? var_name.lower() so it's consistent